Source/CronosPlayUnreal/Public/DefiWalletCoreActor.h

```diff
@@ -189,6 +189,43 @@ class CRONOSPLAYUNREAL_API ADefiWalletCoreActor : public AActor {
     void RestoreWallet(FString mnemonics, FString password, FString &output,
                        bool &success, FString &output_message);
 
+    /**
+     * Restore wallet for secure storage with mnemonics and password (Only for
+     * testing & development purpose).
```

Comment:
if it is from secure storage, why still for testing & development?

Reply:
i'll change the comment